CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ### CHANGELOG
 
+## [v0.21.0]
+- `apiKey` is a new required parameter used for SDK initialization
```

Comment:
Be clear here to reach out to a Lucra contact to obtain the key for your instance